tests/test_shell_completion.py

```diff
@@ -304,3 +304,14 @@ def complete(ctx, param, incomplete):
         env={"COMP_WORDS": "", "COMP_CWORD": "0", "_CLI_COMPLETE": "bash_complete"},
     )
     assert result.output == "plain,a\nplain,b\n"
+
+
+def test_case_insensitive_completions():
+    options = ["Foo", "Bar", "Baz"]
+
+    opts = [Option(["--opt"], type=Choice(options, case_sensitive=False))]
+    cli = Command("cli", params=opts)
+
+    completions = _get_completions(cli, ["--opt"], "f")
+    assert len(completions) == 1
+    assert completions[0].value == "Foo", "expected to find 'Foo' as a completion item"
```

Comment:
Consider adding a test with `case_sensitive=True`

Reply:
Added a case right below this one :)